src/commands/kv/namespace/site.rs

```diff
@@ -1,37 +1,25 @@
-use cloudflare::endpoints::workerskv::create_namespace::{CreateNamespace, CreateNamespaceParams};
-use cloudflare::endpoints::workerskv::list_namespaces::{ListNamespaces, ListNamespacesParams};
 use cloudflare::endpoints::workerskv::WorkersKvNamespace;
-use cloudflare::framework::apiclient::ApiClient;
 use cloudflare::framework::response::ApiFailure;
```

Comment:
the only weirdness here is that, now we are wrapping the basic functionality of cloudflare-rs, which had been my initial approach, but because the function signatures still need to reference the cloudflare-rs package for its types, it's an incomplete wrapper.

Reply:
I feel like this wrapper will be a necessary evil for functions that handle `ApiFailure` with custom logic. Otherwise, callers using the default logic (which typically just prints things out) will not have to deal with this cloudflare-rs logic and can have their API call completely wrapped.